### PR TITLE
Airflow 172, fixes #172

### LINKED ISSUE #172: Make set optional for the OAI intake harvester
Some data providers simply do not implement sets as a feature of their OAI-PMH server. In this case, we can simply use sickle to `ListRecords` and harvest all available records.

### PR DESCRIPTION
**Pull request by jacobthill**

### CODE REVIEW

File: catalogs/qnl.yaml [hunk 1]
```diff
@@ -4,7 +4,9 @@ sources:
   qnl:
     driver: oai_xml
     args:
-      collection_url: https://api.lib.harvard.edu/oai
+      collection_url: https://api.qdl.qa/api/oaipmh
+      metadata_prefix: mods
```
Comment (aaron-collier) [inline catalogs/qnl.yaml]: I think this should be "mods_no_ocr"

**Comment by aaron-collier** [comment]
@jacobthill if you could fix the linting issue I'll approve/merge

File: catalogs/qnl.yaml (hunk 1 above)
Comment (jacobthill) [reply catalogs/qnl.yaml]: Yep, I noticed that right after pushing

**Comment by jacobthill** [comment]
> @jacobthill if you could fix the linting issue I'll approve/merge

Sounds like this is the issue:

dlme_airflow/drivers/oai_xml.py:14:56: W0622: Redefining built-in 'set' (redefined-builtin)

If I understand correctly, there is a built in variable in intake names `set` so I need to remain it here (and in all configs) to something else (perhaps oai_set). Does that sound right to you?

**Comment by aaron-collier** [comment]
@jacobthill no, you should just need to edit the line in the linting suggestion:

https://app.circleci.com/pipelines/github/sul-dlss/dlme-airflow/360/workflows/4e55a2da-ed96-4e70-a5f7-0c08cbf6b9b8/jobs/622

**Review by aaron-collier — APPROVED** [review]

**Pull request merged**